lib/Echidna/Solidity.hs

```diff
@@ -210,7 +211,7 @@ loadSpecified name cs = do
 
   -- Set up initial VM, either with chosen contract or Etheno initialization file
   -- need to use snd to add to ABI dict
-  blank' <- maybe (pure initialVM)
+  blank' <- maybe (pure (initialVM & block . maxCodeSize .~ w256 (fromInteger mcs)))
```

Comment:
maybe we should make `_codeSize` a `Word` instead

Reply:
What’s the reason? I mimicked the way balance works.

Reply:
I think we should merge it and then re-organize the types of the maxCodeSize, balance, etc in another PR